fix_includes_test.py

```diff
@@ -2830,19 +2830,75 @@ def testRemoveDuplicates(self):
 int main() { return 0; }
 """
     iwyu_output = """\
-remove_duplicates should add these lines:
+remove_duplicate_includes should add these lines:
 #include <stdio.h>
 
-remove_duplicates should remove these lines:
+remove_duplicate_includes should remove these lines:
 - #include <notused.h>  // lines 3-3
 
-The full include-list for remove_duplicates:
+The full include-list for remove_duplicate_includes:
 #include <stdio.h>
 #include "used.h"
 #include "used2.h"
 ---
 """
-    self.RegisterFileContents({'remove_duplicates': infile})
+    self.RegisterFileContents({'remove_duplicate_includes': infile})
+    self.ProcessAndTest(iwyu_output)
+
+  def testRemoveDuplicateForwardDeclarations(self):
+    """Tests we uniquify if an #include is in there twice."""
+    infile = """\
+#include <notused.h>  ///-
+class A;
+template<typename T> // Comment in the middle not a problem
+class B;
+class A; ///-
+template<typename T> ///-
+class B; ///-
+template<typename T> class B; ///-
+
+int main() { return 0; }
+"""
+    iwyu_output = """\
+remove_duplicate_forward_declarations should add these lines:
+
+remove_duplicate_forward_declarations should remove these lines:
+- #include <notused.h>  // lines 1-1
+
+The full include-list for remove_duplicate_forward_declarations:
+class A;  // lines 2-2
+template <typename T> class B;  // lines 3-4
+class A;  // lines 5-5
```

Comment:
Why are these duplicates not removed now?

Reply:
They are removed.  That's what the `///-` markings signify in the lines above (in `infile`).

What you're quoting here is IWYU output.  IWYU doesn't do deduping, so it's expected to see the dupes here.